generators/server/templates/src/main/resources/config/application-prod.yml.ejs

```diff
@@ -322,6 +322,7 @@ jhipster:
             key: <%= rememberMeKey %>
 <%_ } _%>
     mail: # specific JHipster mail property, for standard properties see MailProperties
+        enabled: false
```

Comment:
please add docs to the website as this is a change in behavior and people might be confused why emails are disabled